views/account.handlebars

```diff
@@ -0,0 +1,482 @@
+                        <a href="single-listing-1.html" class="image-wrapper background-image">
+                          <img src="assets/img/image-04.jpg" alt="">
+                        </a>
+                      </div>
+                      <!--end image-->
+                      <h4 class="location">
+                        <a href="#">Greeley, CO</a>
+                      </h4>
+                      <div class="price">$75,000</div>
+                      <div class="meta">
+                        <figure>
+                          <i class="fa fa-calendar-o"></i>13.03.2017
+                        </figure>
+                        <figure>
+                          <a href="#">
+                            <i class="fa fa-user"></i>Hills Estate
+                          </a>
+                        </figure>
+                      </div>
+                      <!--end meta-->
+                      <div class="description">
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam venenatis lobortis</p>
+                      </div>
+                      <!--end description-->
+                      <div class="additional-info">
+                        <ul>
+                          <li>
+                            <figure>Area</figure>
+                            <aside>368m<sup>2</sup></aside>
+                          </li>
+                          <li>
+                            <figure>Bathrooms</figure>
+                            <aside>2</aside>
+                          </li>
+                          <li>
+                            <figure>Bedrooms</figure>
+                            <aside>3</aside>
+                          </li>
+                          <li>
+                            <figure>Garage</figure>
+                            <aside>1</aside>
+                          </li>
+                        </ul>
+                      </div>
+                      <!--end addition-info-->
+                      <a href="single-listing-1.html" class="detail text-caps underline">Detail</a>
+                    </div>
+                  </div>
+                   <!--end item-->
+                </div>
+              </div>
+              <!--end container-->
+            </section>
+          <!--end block-->
+          <!--============ End Recent Ads =========================================================================-->
+          <!--============ Newsletter =============================================================================-->
+          
+
+          <section class="block">
+            <div class="container">
+              <div class="d-flex align-items-center justify-content-around">
+              </div>
+            </div>
+          </section>
+
+      </section>
+      <!--end content-->
+
+      <!--*********************************************************************************************************-->
```

Comment:
we could take out this footer as the footer partial is used in the main.handlebars template.